pkgs/applications/version-management/git-and-tools/git-big-picture/default.nix

```diff
@@ -22,7 +22,7 @@ python2Packages.buildPythonApplication rec {
 
   meta = {
     description = "Tool for visualization of Git repositories";
-    homepage = "https://github.com/esc/git-big-picture";
+    homepage = "https://github.com/git-big-picture/git-big-picture";
     license = lib.licenses.gpl3;
```

Comment:
gpl3Plus or gpl3Only?

Reply:
Nice to meet a fellow who cares about licensing too. It's gpl3Plus indeed. Fixed.